plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/user/InviteCommand.java

```diff
@@ -28,11 +39,11 @@ public final class InviteCommand extends AbstractFunnyCommand {
             description = "${user.invite.description}",
             aliases = "${user.invite.aliases}",
             permission = "funnyguilds.invite",
-            completer = "online-players:3",
+            completer = "invite-command",
```

Comment:
Dodaj tutaj `:3`, żeby nie mieć za dużo tab completów

Reply:
Jakieś fanaberie z tym limitowaniem do 3 wyników, podczas gdy wszędzie tak jest i najlepiej zmienić w oddzielnym PR, warto przemyśleć — czy jest sens aż tak to ograniczać? Funkcja staje się przez to mniej użyteczna.

Reply:
Z tym się zgadzam - limitowanie do trzech wyników jest bez sensu

Reply:
Realnie powinien być limit narzucony, jeśli nie jest — przy samym setupie frameworka komend, skoro i tak wszędzie jest 3, to czemu ma być to w każdym miejscu lokalnie. Wartości poniżej 100 to moim zdaniem zbędne ograniczanie.